README.md

```diff
@@ -125,40 +102,22 @@ Here is a tutorial of the current development features:
 		   Some common rates are:  -r 29.97  -r 25  -r 30  -r 60
 
 
-#################
-    WEBSITES
-#################
+## Websites
 
-http://www.openshot.org/  (PROJECT WEBSITE and BLOG)
-http://launchpad.net/openshot/ (SOURCE CODE, BUG TRACKER, TRANSLATIONS)
-http://freshmeat.net/projects/openshot-video-editor/
-https://sourceforge.net/projects/openshotvideo/
-https://www.ohloh.net/p/openshot-video-editor/
+- http://www.openshot.org/  (Official website and blog)
+- http://launchpad.net/openshot/ (source code, bug tracker and translations)
```

Comment:
This, um... should point to Github now, surely? (Launchpad was the project's previous home. The README file is included in the source-release tarballs, so it'll be seen by people who haven't necessarily accessed this repo — that's why self-referential links are still useful.)

...Launchpad can still be linked to _in addition_ to Github, of course, the same way Sourceforge is.

Reply:
Very true. We should be pointing to GitHub now surely.